### PR TITLE
Upgrade Multus CNI

### PR DESCRIPTION
**Pull request by dud225**

### CODE REVIEW

File: addons/multus/multus.yaml [hunk 1]
```diff
@@ -123,17 +136,33 @@ spec:
...
+          resources:
+            requests:
+              cpu: "10m"
+              memory: "15Mi"
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - name: cnibin
+              mountPath: /host/opt/cni/bin
+              mountPropagation: Bidirectional
```
Comment (neoaggelos) [inline addons/multus/multus.yaml:160]: Why is `mountPropagation: Bidirectional`  needed?

**Review by neoaggelos — COMMENTED** [review]
Hi @dud225! Thank you for this PR, I've left a comment

File: addons/multus/multus.yaml (hunk 1 above)
Comment (dud225) [reply addons/multus/multus.yaml:160]: Hello
This setting actually comes from the configuration published by the [upstream project](https://github.com/k8snetworkplumbingwg/multus-cni/blob/8bbb594dad15d8c617fccc21b9409de21686ef40/deployments/multus-daemonset.yml#L211=).
My understanding is that Multus handles the configuration of the network interfaces, as part of that it has to execute various binaries (CNI plugin) depending on the configuration requested, so if a new CNI plugin is installed on Microk8s it has to be immediately available to Multus. At the same time, Multus installs its own binary on this directory [at startup](https://github.com/k8snetworkplumbingwg/multus-cni/blob/8bbb594dad15d8c617fccc21b9409de21686ef40/images/entrypoint.sh#L203-L205=) (I'm not sure of the reason though, maybe this is for consistency) so this mount point needs to be available from both side.

Comment (neoaggelos) [reply addons/multus/multus.yaml:160]: Explanation makes sense, but I would expect the `mountPropagation: Bidirectional` to be used in the volume mount for the container, not the initContainer instead.

Anyway, I did not find anything relevant in https://github.com/k8snetworkplumbingwg/multus-cni/pull/718, so I'm fine with this. Thanks!

**Pull request merged**